src/main/resources/plugin.yml

```diff
@@ -28,6 +28,9 @@ commands:
   ping:
     description: Gets the ping of you or a specified player
     usage: /ping [player]
+  slack:
+    description: Manage the HackCraft Slack integration
+    usage: /slack <link|unlink|lookup> [id|payer]
```

Comment:
Maybe also find a way to clarify that player  is for lookup, Id is for link, and unlink takes no args